falkordb_gemini_kg/steps/extract_data_step.py

```diff
@@ -2,14 +2,13 @@
 from falkordb_gemini_kg.classes.source import AbstractSource
```

Comment:
this step and any other step at our disposal must be well documented
with an Introduction section, Purpose and examples